lib/ProductOpener/GeoIP.pm

```diff
@@ -49,6 +71,22 @@ if (-e $geolite2_path) {
 	$gi = GeoIP2::Database::Reader->new(file => $geolite2_path);
 }
 
+=head1 FUNCTIONS
+
+=head2 get_country_for_ip()
+
+C<get_country_for_ip()> takes the IP address as an input parameter and returns the country name where the IP address is located.
+
+=head3 Arguments
+
+One scalar variable ip is passed as an argument.
+
+=head3 Return values
+
+If the function executes successfully it returns the country name. On the other hand, if it throws an exception, it simply returns undefined.
```

Comment:
It would be good to specify what is the reference for the "country name". I think it's the english version of Geonames.org names but not the ISO 3166-1 -- https://en.wikipedia.org/wiki/ISO_3166-1. The "United Kingdom" for example, seems to be named "United Kingdom of Great Britain and Northern Ireland" in ISO 3166-1. @stephanegigandet could you tell us?